### ISSUE TITLE
Rework OEO pages: OEO-Viewer, Class-Viewer

### ISSUE THREAD

**Issue by jh-RLI** [OpenEnergyPlatform/oeplatform#1911]
## Description of the issue

We currently offer several OEO related features on the OEP. The OEO-Viewer to browse the contents, the OEO-Classes to provide a URI for each Class to make the OEO searchable & its Content linkable. We noticed that among all the other goals we are working on, maintaining all these different tools is too much. 

A better way is to collaborate with other developers who are currently working on tools to provide insights to Ontologies via the WEB.

## Ideas of solution

1. The TIB offers all tools that we want to browse a ontology:
  - Link to the TIB to browse the OEO
  - Link to tib´s WEBVOWL Graph viewer
  - Remove the OEO-Viewer
2. The OEO Classes page should be extended.
  - Rework the page layout
  - Extent the page content ( similar to https://terminology.tib.eu/ts/ontologies/oeo/terms)

## Workflow checklist

- [x] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md)

**Comment by stap-m** [comment]
Good point. Would you present these ideas in one of the next oeo-dev-meetings?

**Comment by maurerle** [comment]
I'd like to share this feature of the OEO-Viewer here:

https://github.com/user-attachments/assets/f1b8e73c-5afd-4fe8-8d55-326f23f62818

Looks like a submission to https://www.reddit.com/r/badUIbattles/ :)
Would be sad to see this gone ;) though I think that other owl viewers would indeed help here

**Comment by jh-RLI** [comment]
Haha, that is true ;) It is only a prototype, from some years ago. But i also see why it is this as it seems to be quite a lot of work to implement a good ui here. 

https://service.tib.eu/webvowl/#iri=https://openenergyplatform.org/ontology/oeo/releases/oeo-full.owl
Is much better.

**Comment by jh-RLI** [comment]
But even [webvvowl ](https://github.com/VisualDataWeb/WebVOWL) (latest update is from 2019) is not actively maintained anymore. Actually there is a lack of a modern tool to tackle tasks around viewing, searching and editing ontologies in the browsers. I just came across this link where someone wants to create a new product https://xvowl.raydak.de/

**Comment by jh-RLI** [comment]
We team up with TS4NFDI and take part in one of their incubator projects. They developed a nice suite that offers modular widgets we can use. They also offer complete services that would fit our needs.

https://ts4nfdi.github.io/terminology-service-suite/comp/latest/?path=/docs/overview--docs

**Comment by han-f** [comment]
I agree strongly that a rework of the OEO classes viewer (I think this is what I called textual viewer in SIROP) could add a lot of value added. Including a search bar would already be a large improvement.

**Comment by jh-RLI** [comment]
I will reopen this until the class viewer page is also reworked.
